kolibri_instant_schools_plugin/assets/src/styles/theme.styl

```diff
@@ -24,7 +23,7 @@ $app-bar-title-icon = url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAA
 */
 
 $core-font = 'NotoSans', 'sans-serif'
-
+$core-action-light = #E2D1E0
```

Comment:
![image](https://user-images.githubusercontent.com/9877852/32584445-25d85d08-c4ad-11e7-863d-88570d4cdea4.png)
@christianmemije

Reply:
this is being set to the value in [Kolibri's theme file](kolibri/core/assets/src/styles/core-theme.styl). Every variable defined there has to be defined here.